### DISCUSSION TITLE
How to clear database table for tests

### DISCUSSION THREAD

**Discussion by ashramsey** [jobrunr/jobrunr#506] [Q&A] [answered]
Hi, I was wondering how people clear the jobrunr_jobs table in their tests. I was looking for a deleteAll() call on BackgroundJob.

Thanks

**Comment by rdehuyss** [comment, marked as answer]
You could use the InMemoryStorageProvider and create a new one before each test?

**Reply by ashramsey** [reply]
Ah cool, thanks so much for getting back to me. I'll take a look.

I got around it by creating a jdbc crud repository for the table and calling the deleteAll() in the test setup/teardown.
